unit_tests/schema_tests.json

```diff
@@ -0,0 +1,579 @@
+          "description": "valid date"
+        },
+        {
+          "payload": "1/2/1975",
+          "expected": false,
+          "description": "invalid date"
+        }
+      ]
+    },
+    "gender":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/gender",
+      "payload_tests": [
+        {
+          "payload": "Male",
+          "expected": true,
+          "description": "check 'Male' is ok"
+        },
+        {
+          "payload": "Female",
+          "expected": true,
+          "description": "check 'Female' is ok"
+        },
+        {
+          "payload": "Not Specified",
+          "expected": true,
+          "description": "check 'Not Specified' is ok"
+        },
+        {
+          "payload": "something else",
+          "expected": false,
+          "description": "other values are not OK"
+        },
+        {
+          "payload": "",
+          "expected": false,
+          "description": "blank is not ok"
+        }
+      ]
+    },
+    "phone_number":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/phone_number",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "07234243401",
+          "expected": true,
+          "description": "valid phone numbers start with 07"
+        },
+        {
+          "payload": "082342323401",
+          "expected": false,
+          "description": "does not start with 07 (but right length)"
+        },
+        {
+          "payload": "072342334",
+          "expected": false,
+          "description": "starts with 07 but too short"
+        },
+        {
+          "payload": "082342323143",
+          "expected": false,
+          "description": "starts with 07 but too long"
+        },
+        {
+          "payload": "0723 4243401",
+          "expected": false,
+          "description": "starts with 07 but has a space"
+        },
+        {
+          "payload": "0723 424340",
+          "expected": false,
+          "description": "starts with 07 but has a space"
+        }
+      ]
+    },
+    "address":{
+      "pattern_path": "/definitions/PrivateIndividualName/properties/address",
+      "payload_tests": [
+        {
+          "payload": "",
+          "expected": false,
+          "description": "must not be blank"
+        },
+        {
+          "payload": "PL21 0AQ",
+          "expected": true,
+          "description": "valid post code"
+        },
+        {
+          "payload": "PL21 0CC",
+          "expected": false,
+          "description": "invalid end characters"
+        },
+        {
+          "payload": "PL 0AQ",
+          "expected": false,
+          "description": "invalid start of code"
+        },
+        {
+          "payload": "PL21 0KK",
+          "expected": false,
+          "description": "invalid end characters"
+        },
```

Comment:
Any need for this scenario as it seems to be just a duplicate of the one above using postcode PL21 0CC.